src/main/resources/logback-spring.xml

```diff
@@ -71,5 +71,9 @@
         <root level="info">
             <appender-ref ref="stdout" />
         </root>
+
+        <logger name="uk.gov.justice.digital.hmpps.hmppsinterventionsservice" level="${LOGGING_LEVEL_UK_GOV_JUSTICE_DIGITAL_HMPPS_HMPPSINTERVENTIONSSERVICE:-info}" additivity="false">
```

Comment:
you shouldn't have to add this - i tried it out and it worked with just the env var for me

Reply:
oh that's awsome, thanks